### ISSUE TITLE
say-as - format property is not correctly validated on the Google-platform

### ISSUE THREAD

**Issue by janniksam** [gsdriver/ssml-check#5]
According to google documenation:

> The format attribute is a sequence of time field character codes. Supported field character codes in format are {h,m, s, Z, 12, 24} for hour, minute (of the hour), second (of the minute), time zone, 12-hour time, and 24-hour time respectively. If the field code appears once for hour, minute, or second then the number of digits expected are 1, 2, and 2 respectively. If the field code is repeated then the number of expected digits is the number of times the code is repeated. Fields in the time text may be separated by punctuation and/or spaces. If hour, minute, or second are not specified in the format or there are no matching digits then the field is treated as a zero value. The default format is "hms12".

The validation only does allow some of the possibilities offered by Google:

```js
} else if (attribute === 'format') {
              if (['mdy', 'dmy', 'ymd', 'md', 'dm', 'ym',
                  'my', 'd', 'm', 'y'].indexOf(element.attributes.format) === -1) {
                errors.push(createTagError(element, attribute));
}
```

**Comment by gsdriver** [comment]
It also looks like format is not supported for "time" on Alexa; so this additional check of a proper time format string should only happen on Google.

**Comment by gsdriver** [comment]
Fixed in version 0.2.3
